applications/launchpad_v2/src/components/TitleBar/index.tsx

```diff
@@ -0,0 +1,208 @@
+            data-testid='minimize-window-btn'
+          >
+            <svg
+              xmlns='http://www.w3.org/2000/svg'
+              width='10'
+              height='2'
+              viewBox='0 0 10 2'
+              fill='none'
+              style={{
+                opacity: 0,
+              }}
+            >
+              <path
+                d='M1 1H9'
+                stroke='#C2903A'
+                strokeWidth='1.5'
+                strokeLinecap='round'
+              />
+            </svg>
+          </TitleBarButton>
+          <TitleBarButton
+            borderColor='#51A73E'
+            background='#61C354'
+            style={{ padding: 0 }}
+            onClick={onMaximize}
+            data-testid='maximize-window-btn'
+          >
+            <svg
+              xmlns='http://www.w3.org/2000/svg'
+              width='17'
+              height='16'
+              viewBox='0 0 17 16'
+              fill='none'
+              style={{
+                opacity: 0,
+              }}
+            >
+              <path
+                d='M4.04504 4.32699C4.04331 3.99321 4.31434 3.72219 4.64812 3.72391L9.99044 3.75145C10.5235 3.7542 10.7885 4.39878 10.4116 4.77571L5.09683 10.0905C4.7199 10.4674 4.07532 10.2024 4.07257 9.66932L4.04504 4.32699Z'
+                fill='#407C33'
+              />
+              <path
+                d='M11.7442 12.0263C12.078 12.028 12.349 11.757 12.3473 11.4232L12.3197 6.08085C12.317 5.5478 11.6724 5.28275 11.2955 5.65968L5.98068 10.9745C5.60376 11.3514 5.86881 11.996 6.40185 11.9987L11.7442 12.0263Z'
+                fill='#407C33'
+              />
+            </svg>
+          </TitleBarButton>
+        </WindowButtons>
+
+        <LogoContainer style={{ ...logoColorAnim }}>
+          <Logo variant='full' />
+        </LogoContainer>
+      </LeftCol>
+
+      <div
+        data-tauri-drag-region
+        style={{
+          width: '100%',
+          height: '100%',
+          overflow: 'hidden',
+          zIndex: 1,
+          position: 'absolute',
+          top: 0,
+          left: 0,
+          right: 0,
+          bottom: 0,
+          borderTopRightRadius: 10,
+        }}
+      >
+        <animated.div
+          data-tauri-drag-region
+          style={{
+            position: 'absolute',
+            top: 0,
+            bottom: 0,
+            right: 0,
+            background: '#2a2a2a',
+            borderTopRightRadius: 10,
+            borderTopLeftRadius: expertViewSize === '100%' ? 10 : 0,
+            ...drawerContainerStyle,
+          }}
+        />
+      </div>
+      <animated.div
+        style={{
+          position: 'absolute',
+          top: 0,
+          right: 16,
+          padding: 16,
+          zIndex: 12,
+          display: 'flex',
+          height: '100%',
+          alignItems: 'center',
+          boxSizing: 'border-box',
+        }}
+      >
+        <Button variant='text' leftIcon={<SvgSetting />}>
+          Settings
```

Comment:
we agreed that texts visible on the screen should be extracted to a separate file to be ready for i18n